modules/developer_manual/pages/general/security.adoc

```diff
@@ -153,7 +153,7 @@ because all `$_REQUEST` parameters are strings. However, if you expect
 text as a parameter, use
 link:http://php.net/manual/en/function.htmlspecialchars.php[PHP’s htmlspecialchars function] with 
 `ENT_QUOTES` or `strip_tags` to prevent
-link:https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)[Cross-site Scripting (XSS) attacks].
+link:++https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)++[Cross-site Scripting (XSS) attacks].
```

Comment:
What's with the surrounding `+`'s?

Reply:
Okay, I see the issue after experimenting with it locally. Strange that it should without the plus signs, it would cause the paragraph to italicise.